modules/aws-eks-managed-node-groups/variables.tf

```diff
@@ -79,6 +79,6 @@ variable "http_tokens" {
 
 variable "http_put_response_hop_limit" {
   type        = number
-  default     = 1
+  default     = 2
```

Comment:
Fix for https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.3/deploy/installation/#using-metadata-server-version-2-imdsv2